apps/parsley/package.json

```diff
@@ -24,7 +24,7 @@
     "prod": "env-cmd -e devProduction -r .env-cmdrc.local.json vite",
     "eslint:fix": "yarn eslint:strict --fix",
     "eslint:staged": "STRICT=1 eslint",
-    "eslint:strict": "STRICT=1 eslint '*.{js,ts,tsx}' 'src/**/*.ts?(x)' 'cypress/**/*.ts' 'src/gql/**/*.graphql'",
+    "eslint:strict": "STRICT=1 eslint .",
```

Comment:
When I run `eslint:strict`, I get this 😅:

<img width="544" alt="Screenshot 2024-04-26 at 4 50 13 PM" src="https://github.com/evergreen-ci/ui/assets/47064971/ba0c782c-c519-4ae6-a17b-6eec6dbe1582">


I have a feeling that it's also scanning my files in the `/dist`, `/bin` folders etc and that we might need to exclude them